package/upgrade/upgrade_manifests.sh

```diff
@@ -1277,7 +1277,7 @@ EOF
   fi
 }
 
-apply_whereabouts_manifests()
+apply_extra_nonversion_manifests()
```

Comment:
here strict version control is required, to avoid downgrade CRDs accidentally

Reply:
I thought the CRD we updated here was the non-handled CRD (which usually happened on the upgrade). I thought we would not encounter the downgrade issue here.
cc @starbops, @WebberHuang1118 WDYT?

Reply:
The concern comes from the long-time version management, today we all remember the cdi has been put on 2 places. Later, we or others may forget to bump the two places in parallel (surely, with lower chance but it could happen), and it may took additional efforts to figure them out.

```
harvester version              v1.5        v1.6   v1.7   v1.8

upgrade path                   v1.0        v1.0   v1.0   ?
harvester chart path           v1.0        v1.0   v1.1   v1.2
```

Reply:
On other parts, files are managed per version, e.g.

https://github.com/harvester/harvester/blob/ec25120bade7191e583a8298c600b1a803112824/package/upgrade/migrations/upgrade_manifests/v1.4.0/pre-hook.sh#L3